source/quickstart.txt

```diff
@@ -1,12 +1,235 @@
-===========
+==========
+Quickstart
+==========
 
 .. include:: /includes/preview-warning
 
 .. default-domain:: mongodb
 
-Get going!
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+{+c2c-full-product-name+} connects MongoDB clusters and provides a way
+to synchronize data between them. The tool that makes the connection is
+:ref:`c2c-mongosync`. This page provides a short introduction to help
+you get started with {+c2c-product-name+}. For more details, please see
+the rest of the {+c2c-product-name+} documentation.
+
+Setup
+-----
+
+{+c2c-product-name+} synchronizes data between two clusters. To run
+``mongosync``, you will need to: 
+
+- Define a source and a destination cluster.
+- Define administrative users 
+- Download and install ``mongosync``.
+- Connect the clusters.
+
+Follow the instructions below to setup {+c2c-product-name+}.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Define a source and a destination cluster
+
+      If you already have a MongoDB cluster, either self-managed or
+      hosted in :atlas:`MongodDB Atlas </getting-started?jmp=docs>`, 
+      use that cluster as the source cluster. If you don't have a
+      cluster to work with, you will need to create one.
+
+      You will need a destination cluster that has the same topology
+      as your source cluster. If the source cluster is :ref:`sharded
+      <sharding-background>`, the destination cluster must be sharded
+      in the same way. If the source cluster is a :ref:`replica set
+      <replication>` with 3 nodes, the destination cluster must also be
+      a replica set with 3 nodes.
+
+      The source and destination clusters must be at least MongoDB 6.0.
+      If your clusters are self-managed, they must be MongoDB
+      Enterprise clusters. {+c2c-product-name+} is not supported on
+      MongoDB Community Edition.
+
+      Creating a cluster is beyond the scope of this guide. If you need
+      help, refer to the documentation to create an :ref:`Atlas cluster
+      <create-new-cluster>` or to create a :ref:`self-managed cluster
+      <server-replica-set-deploy>`.
+
+   .. step:: Define administrative users
+
+      If either cluster is hosted in Atlas, or if either of them
+      requires authentication, you must create a database user that has
+      permissions in both clusters. 
+
+      Source Cluster Authentication Requirements
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      The source user must have the following roles:
+
+      - :authrole:`readAnyDatabase` role
+      - :authrole:`clusterMonitor` role
+      - :authrole:`backup` role
+
+      In addition, the source user must be able to:
+
+      - Run the :dbcommand:`getParameter` command
+
+      If the source cluster is hosted in Atlas, the user must have the
+      :ref:`Atlas admin <atlas-user-privileges>` role. The user must
+      also be able to :ref:`read the oplog <ref-atlas-oplog>`.
```

Comment:
`mongosync` uses change streams, not oplog. Should we mention something like [this](https://www.mongodb.com/docs/v6.0/changeStreams/#access-control) here? I think `mongosync` would read the change stream for the entire cluster.